src/app/shared/entry/descriptor-language.pipe.ts

```diff
@@ -0,0 +1,24 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DescriptorLanguageService } from 'app/shared/entry/descriptor-language.service';
+import { Workflow } from '../../shared/swagger';
+
+@Pipe({
+  name: 'descriptorLanguage',
+})
+export class DescriptorLanguagePipe implements PipeTransform {
+  /**
+   * When displaying the descriptor type typically the workflow descriptor type string is used
+   * However for Galaxy this is 'gxformat2', so change it to be Galaxy.
+   * The shortFriendlyName is 'Galaxy' so use that.
+   * @param descriptorType
+   * @returns {string}
+   */
+  constructor(private descriptorLanguageService: DescriptorLanguageService) {}
+  transform(descriptorType: string): string {
+    if (descriptorType === 'gxformat2') {
+      return this.descriptorLanguageService.workflowDescriptorTypeEnumToShortFriendlyName(Workflow.DescriptorTypeEnum.Gxformat2);
+    } else {
+      return descriptorType.toUpperCase();
```

Comment:
Do we uppercase everything else because it's mostly acronyms (CWL, WDL, NFL)?   I wonder if we should use the "short" friendly name for all cases?  Or, create another field for the "abbreviated" friendly name in `ExtendedDescriptorLanguageBean` and use that?

Reply:
Yeah everything else should be acronyms, I think the only additional difference would be using `Nextflow` instead of `NFL`. (and `SMK` to `Snakemake`) @denis-yuen any opinion?
![image](https://user-images.githubusercontent.com/97123241/219060016-d1b435d5-4547-4257-ab07-d60928cff4bc.png)
![image](https://user-images.githubusercontent.com/97123241/219062704-2ee0311d-6dda-4c37-8429-3b3bfb810fee.png)

Reply:
Nextflow has approved NFL as an acronym, no idea about Snakemake. 
But from and end-user POV, Nextflow and Snakemake are probably better than the acronyms for those two languages, so:
`CWL, WDL, Galaxy, Nextflow, Snakemake` could work given that there's plenty of space on this screen. (There is less on say, the search page)

Reply:
Nextflow is only _slightly_ bigger than Galaxy on the search page, but I agree readability is better than the acronyms... I'll switch them all to friendly names.

![image](https://user-images.githubusercontent.com/97123241/219070913-a67c40f0-016e-4ae9-afe8-c6e66aa338c6.png)